README.md

```diff
@@ -139,6 +139,7 @@ These themes will be installed in a default installation.
 
 - [bragi](https://github.com/eduNEXT/ednx-saas-themes/tree/edunext/mango.master)
 
+You can now set the default theme you choose. When wiritng the strian.yml file, the defautl theme will be the first at the DISTRO_THEMES_NAME argument.
```

Comment:
@danielsgz please correct the description and change ``strain.yml`` to ``config.yml``, not all developers use strain.yml

Reply:
Noted and corrected @JuanDavidBuitrago